install/install_utils.php

```diff
@@ -2169,3 +2169,66 @@ function enableInsideViewConnector()
     // $mapping is brought in from the mapping.php file above
     $source->saveMappingHook($mapping);
 }
+
+/**
+ * Gets the username of the user under which the PHP script is currently running
+ * Notes:
+ * - works on Windows and Linux, tries a variety of methods to accomodate different systems and hosting restrictions
+ * - on Windows, return full username in form DOMAIN\USER
+ * - returns empty string if failed
+*/
+
+function getRunningUser()
+{
+    // works on Windows and Linux, but might return null on systems that include exec in disabled_functions in php.ini (typical in shared hosting)
+    $runningUser = exec('whoami');
+
+    if ($runningUser == null) {  // matches null, false and ""
+        if (is_windows())
```

Comment:
Sorry - my mistake....I didn't examine it closely enough as I was only looking at the diffs. I was thinking about the uniformity of using:
posix_getpwuid(posix_geteuid())['name'];
and avoiding the use of exec('whoami')